runtime/autoload/health/nvim.vim

```diff
@@ -3,9 +3,20 @@ let s:bad_responses = [
             \ 'unable to parse python response',
             \ 'unable to parse',
             \ 'unable to get pypi response',
-            \ 'unable to get neovim executable'
+            \ 'unable to get neovim executable',
+            \ 'unable to find neovim version'
```

Comment:
Or there were cases where we returned something like this `[neovim_version, "unknown", other_var]` and it wouldn't be clear what the `unknown` meant.